tiatoolbox/utils/visualization.py

```diff
@@ -851,11 +851,41 @@ def render_pt(
                 top_left,
                 scale,
             )[0][0],
-            np.maximum(self.edge_thickness, 1),
+            np.maximum(int(16 / scale**0.5), 1),
             col,
-            thickness=self.thickness,
+            thickness=-1,
```

Comment:
why change self.thickness to a static value?

Reply:
The reason for these changes was to make points be more consistently visible across zoom levels. The thickness=-1 specifically is so that the point is filled. self.thickness value is for the thickness of boundary lines around polygons, but as a point-like object without a boundary, the value doesn't make sense to be used here.